src/types.ts

```diff
@@ -57,25 +57,42 @@ export type DidDeclineAllHandler = (opts?: {
   includingNecessary: boolean;
 }) => boolean;
 
-export interface CookieConsentHookActions {
+export interface ConsentHookActions {
   acceptCookies: (cookies: CookieTypes, options?: AcceptCookiesOptions) => void;
```

Comment:
This function is also unique for the cookie storage, for the other one I'd remove the `options`